src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -217,8 +235,11 @@ class certificates extends viewModelBase {
     }
     
     enterEditCertificateMode(itemToEdit: unifiedCertificateDefinition) {
-        this.model(certificateModel.fromDto(itemToEdit));
-        this.model().validationGroup.errors.showAllMessages(false);
+        if (!_.includes(itemToEdit.Thumbprints, this.serverCertificateThumbprint())) {
```

Comment:
do we need this check here? I think we should just disable button?

Reply:
removed